test/test_testing.py

```diff
@@ -250,13 +278,8 @@ class MyTestingCharm(CharmBase):
         def cleanup():
             sys.path = orig
             sys.modules.pop('charm')
+
         self.addCleanup(cleanup)
```

Comment:
```suggestion
        self.addCleanup(cleanup)

    def test_actions_passed_in(self):
        harness = Harness(CharmBase, meta='''
            name: test-app
        ''', actions='''
            test-action:
              description: a dummy test action
        ''')
        self.assertEqual(['test-action'], list(harness.framework.meta.actions))
```

Reply:
Reading through that, I'm not a huge fan of how it reads, maybe:
```
        harness = Harness(
            CharmBase,
            meta='''
                name: test-app
        ''',
            actions='''
                test-action:
                    description: foo
        ''')
```

Seems the clearest as to conveying what is what.